config/locales/en.yml

```diff
@@ -30,4 +30,139 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  confirmation:
```

Comment:
for each section could be a different file (`config/locales/en/confirmation.yml`), it's easier to support